package.json

```diff
@@ -229,7 +229,7 @@
     "js-yaml": "^3.13.1",
     "klaw": "^3.0.0",
     "livepush": "^3.5.0",
-    "lodash": "^4.17.15",
+    "lodash": "^4.17.19",
```

Comment:
Updated this to dedupe deps but it also includes a security issue, I do not believe the security issue affects us but there's no reason not to update